Rollbar.App.Config/RollbarConfigurationLoader.cs

```diff
@@ -0,0 +1,63 @@
+﻿namespace Rollbar.App.Config
+{
+    using Rollbar;
+    using Rollbar.NetStandard;
+    using Rollbar.Telemetry;
+
+    /// <summary>
+    /// Class RollbarConfigurationLoader.
+    /// Implements the <see cref="Rollbar.NetStandard.IRollbarConfigurationLoader" />
+    /// </summary>
+    /// <seealso cref="Rollbar.NetStandard.IRollbarConfigurationLoader" />
+    public class RollbarConfigurationLoader
+        : IRollbarConfigurationLoader
+    {
+        /// <summary>
+        /// Loads the povided configuration object based on found configuration store (if any).
+        /// </summary>
+        /// <param name="config">The configuration.</param>
+        /// <returns><c>true</c> if configuration was found, <c>false</c> otherwise.</returns>
+        public bool Load(RollbarConfig config)
+        {
+            return AppConfigUtility.LoadAppSettings(config);
+        }
+
+        /// <summary>
+        /// Loads the povided configuration object based on found configuration store (if any).
+        /// </summary>
+        /// <param name="config">The configuration.</param>
+        /// <returns><c>true</c> if configuration was found, <c>false</c> otherwise.</returns>
+        public bool Load(TelemetryConfig config)
+        {
+            return AppConfigUtility.LoadAppSettings(config);
+        }
+
+        /// <summary>
+        /// Loads the rollbar configuration.
+        /// </summary>
+        /// <returns>IRollbarConfig or null if no configuration store was found.</returns>
+        public IRollbarConfig LoadRollbarConfig()
+        {
+            RollbarConfig config = new RollbarConfig("seedToken");
```

Comment:
What does "seedToken" mean here? Should it be a configuration value? Or maybe a constant instead of a hard-coded string?

Reply:
it is a localized development trick. i just find it easier to troubleshoot any dynamic configuration failures if/when they happen in a debugger.